### ISSUE TITLE
Failed to activate the php-namespace-monkey package

### ISSUE THREAD

**Issue by s7sundar** [underground-works/atom-php-namespace-monkey#1]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.20.1 x64
**Electron**: 1.6.9
**OS**: Ubuntu 16.04.3
**Thrown From**: [php-namespace-monkey](https://github.com/underground-works/atom-php-namespace-monkey) package 0.1.0

### Stack Trace

Failed to activate the php-namespace-monkey package

```
At Cannot read property 'psr-0' of undefined

TypeError: Cannot read property 'psr-0' of undefined
    at /packages/php-namespace-monkey/lib/php-namespace-monkey.js:101:57
    at Array.forEach (native)
    at /packages/php-namespace-monkey/lib/php-namespace-monkey.js:100:34
    at Array.forEach (native)
    at Object.loadNamespaces (/packages/php-namespace-monkey/lib/php-namespace-monkey.js:87:33)
    at Object.activate (/packages/php-namespace-monkey/lib/php-namespace-monkey.js:39:14)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app/src/package.js:253:25)
    at /usr/share/atom/resources/app/src/package.js:225:38
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app/src/package.js:99:21)
    at /usr/share/atom/resources/app/src/package.js:218:32
    at Package.module.exports.Package.activate (/usr/share/atom/resources/app/src/package.js:215:40)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app/src/package-manager.js:645:40)
    at /usr/share/atom/resources/app/src/package-manager.js:626:35
    at Config.module.exports.Config.transactAsync (/usr/share/atom/resources/app/src/config.js:367:24)
    at PackageManager.module.exports.PackageManager.activatePackages (/usr/share/atom/resources/app/src/package-manager.js:621:25)
    at PackageManager.module.exports.PackageManager.activate (/usr/share/atom/resources/app/src/package-manager.js:603:52)
    at /usr/share/atom/resources/app/src/atom-environment.js:843:36
```

### Commands

```
```

### Non-Core Packages

```
atom-autocomplete-php 0.25.6 
open-from-namespace 0.3.2 
php-namespace-monkey 0.1.0 
```

Help me how to resolve this

**Comment by lsalomon** [comment]
that is a great idea for a plugin, but I got the same issue :(

**Comment by itsgoingd** [comment]
Hey, just published 0.2 with fix for this and couple of other issues.

**Comment by lsalomon** [comment]
Thanks a lot for your quick support !!!

Like you said... This plugin enables us to save a massive amount of time every day !!!

Cheers mate !!
